mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -188,6 +189,21 @@ def predictions_to_json(raw_predictions, output):
     json.dump(predictions, output, cls=NumpyEncoder)
 
 
+def load_predictions_from_json_str(json_str):
+    raw_predictions = json.loads(json_str)  # TODO: Implement numpy decoder and set cls to be numpy decoder ?
```

Comment:
Why do we need to implemente numpy?

Reply:
Because in `predictions_to_json` function, it does `json.dump(predictions, output, cls=NumpyEncoder)` which using `NumpyEncoder`, so when decoding we need a corresponding decoder ? But I am not very sure whether we need it.

Reply:
We don't because we don't need to convert a list to a numpy array.